ReleaseHistory.md

```diff
@@ -1,4 +1,8 @@
 # Microsoft.Security.Utilities Release History
+
+## **v1.5.4** [NuGet Package](https://www.nuget.org/packages/Microsoft.Security.Utilities/1.5.4)
```

Comment:
Any particular reason why we are maintaining this information separately in `ReleaseHistory`, when it can merged into `ReleaseNotes`?